docs/guides/choose-execution-mode.mdx

```diff
@@ -0,0 +1,76 @@
+   - Useful for workloads that don’t have all inputs ready at the outset, for iterative workloads that require classical post-processing before the next one can run, and for experiments that need to run as tightly together as possible.
+- **Job**
+   - Easiest to use when running a small experiment.
+   - Might run sooner than batch mode.
+
+<span id="best-practices"></span>
+## Recommendations and best practices
+
+Generally, use batch mode unless you have workloads that don’t have all inputs ready at the outset.
+
+- Use **batch** mode to submit multiple primitive jobs simultaneously to shorten processing time.
+- Use **session** mode for iterative workloads, or if you need dedicated access to the QPU.
+- Always use **job** mode to submit a single primitive request.
+- Because _sessions are generally more expensive_, it is recommended that you use batch whenever you don't need the additional benefits from using sessions.
+
+To ensure the most efficient use of the execution modes, the following practices are recommended:
+
+- There is a fixed overhead associated with running a job. In general, if each of your jobs uses less than one minute of _QPU time_, consider combining several into one larger job (this applies to all execution modes). "QPU time" refers to time spent by the QPU complex to process your job.
+
+- If each of your jobs consumes more than one minute of QPU time, or if combining jobs is not practical, you can still run multiple jobs in parallel. Every job goes through both classical and quantum processing. While a QPU can process only one job at a time, up to five classical jobs can be processed in parallel. You can take advantage of this by submitting multiple jobs in [batch](/guides/run-jobs-batch#partition) or [session](/guides/run-jobs-session#two-vqe) execution mode.
+
+The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, if you are using a session to get exclusive access to a backend, consider breaking up large jobs into smaller ones and running them in parallel. This might be more cost-effective because it can reduce wall clock time.
+
+## Examples
+
+### Run a quantum variational algorithm
+
+Running a quantum variational algorithm typically follows this flow:
+
+1. Prepare the ansatz.
+2. Evaluate the cost function on a QPU.
+3. Take the result from the previous step and run it through a classical optimizer.
+4. Adjust the parameters according to the output of (3), then go back to step (2).
+
+In this case, if you were using job or batch mode, each job generated by step (2) needs to go back through the queue. This drastically increases the experiment length (wall clock time) due to the queuing time. It could also take longer to converge due to device drift.  That is, every iteration is supposed give you a better result, but device drift could make subsequent results worse.
```

Comment:
```suggestion
In this case, if you were using job or batch mode, each job generated by step (2) needs to go back through the queue. This drastically increases the experiment length (wall-clock time) due to the queuing time. It could also take longer to converge due to device drift.  That is, every iteration is supposed give you a better result, but device drift could make subsequent results worse.
```